src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -276,51 +297,63 @@ private static void validateTableAccess(Client engineClient, JSONObject mapping,
         boolean shouldCheckStreaming = streamingEnabled;
 
         try {
-            if (shouldCheckStreaming && isStreamingPolicyEnabled(DATABASE, database, engineClient, database)){
+            if (shouldCheckStreaming && isStreamingPolicyEnabled(DATABASE, database, engineClient, database)) {
                 shouldCheckStreaming = false;
             }
             try {
-                KustoOperationResult rs = engineClient.execute(database, String.format(FETCH_TABLE_COMMAND, table), validateOnlyClientRequestProperties);
+                KustoOperationResult rs = engineClient.execute(database, String.format(FETCH_TABLE_COMMAND, table),
+                        validateOnlyClientRequestProperties);
                 if (VALIDATION_OK.equals(rs.getPrimaryResults().getData().get(0).get(0))) {
                     hasAccess = true;
                 }
             } catch (DataServiceException e) {
-                databaseTableErrorList.add(String.format("Couldn't validate access to Database '%s' Table '%s', with exception '%s'", database, table, ExceptionUtils.getStackTrace(e)));
+                databaseTableErrorList
+                        .add(String.format("Couldn't validate access to Database '%s' Table '%s', with exception '%s'",
+                                database, table, ExceptionUtils.getStackTrace(e)));
             }
 
             if (hasAccess && StringUtils.isNotEmpty(mappingName)) {
                 try {
-                    engineClient.execute(database, String.format(FETCH_TABLE_MAPPING_COMMAND, table, format.toLowerCase(Locale.ROOT), mappingName));
+                    engineClient.execute(database, String.format(FETCH_TABLE_MAPPING_COMMAND, table,
+                            format.toLowerCase(Locale.ROOT), mappingName));
                 } catch (DataServiceException e) {
                     hasAccess = false;
-                    databaseTableErrorList.add(String.format("Database:%s Table:%s | %s mapping '%s' not found, with exception '%s'", database, table, format, mappingName, ExceptionUtils.getStackTrace(e)));
+                    databaseTableErrorList
+                            .add(String.format("Database:%s Table:%s | %s mapping '%s' not found, with exception '%s'",
+                                    database, table, format, mappingName, ExceptionUtils.getStackTrace(e)));
                 }
             }
 
             if (hasAccess) {
-                String authenticateWith = String.format("aadapp=%s;%s", config.getAuthAppid(), config.getAuthAuthority());
+                String authenticateWith = String.format("aadapp=%s;%s", config.getAuthAppid(),
```

Comment:
Unfortunately we enforced a formatter in the previous revision. Could you merge with master and run mvn formatter:format